### PR TITLE
companion - 0.9.0

### PR DESCRIPTION
**Pull request by smohan-dw**
companion of https://github.com/dhiway/cord/pull/256 

Updates augument-api, and score module.

### CODE REVIEW

**Comment by amarts** [comment]
@smohan-dw please check if the changes are OK now

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "yarn workspaces foreach -Apt --exclude '{root-workspace}' run build",
     "build:docs": "typedoc --skipErrorChecking --theme default --out docs/api --tsconfig tsconfig.docs.json && touch docs/.nojekyll",
     "bundle": "yarn workspace @cord.network/sdk run bundle",
-    "clean": "rimraf tests/dist && yarn workspaces foreach -p --exclude '{root-workspace}' run clean",
+    "clean": "rimraf tests/dist && yarn workspaces foreach -Ap --exclude '{root-workspace}' run clean",
```
Comment (amarts) [inline package.json:12]: this is required in `prepublish`, `publish`, and `publish-beta` options too.

**Review by amarts — APPROVED** [review]

**Comment by amarts** [comment]
I pushed the previous change which had conflicts to https://github.com/amarts/cord.js/tree/sm-updates-0x3_v1 (for your reference if something got missed out).

**Review by smohan-dw — COMMENTED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (smohan-dw) [reply package.json:12]: True, will send a separate PR for that

**Pull request merged**